### DISCUSSION TITLE
Removal of repetitions

### DISCUSSION THREAD

**Discussion by furgon777** [WinMerge/winmerge#1537] [Q&A]
Good afternoon, dear friends, I encountered such a problem: when mass comparing 2 folders with txt files, there are always differences on both sides. How do I bulk remove all differences between 2 files (I mean not move differences from one file to another, namely remove differences from file A and B)
For example as below.
<img width="320" alt="123" src="https://user-images.githubusercontent.com/116534424/197441967-ea29cf48-3359-4628-9558-c1cd1b0c0fb9.png">

**Comment by sdottaka** [comment]
It is possible to remove differences from both files as follows.

![#1537](https://user-images.githubusercontent.com/98126/197556223-b9a81503-5bee-4982-ad83-587230f55333.gif)

Unfortunately, there is no way to do this automatically for large numbers of files.
